local_testing/run_bash_shell_tests.sh

```
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Ensure we're in the correct directory
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)